src/wp-includes/class-wp-script-modules.php

```diff
@@ -405,61 +522,54 @@ public function print_import_map() {
 	 */
 	private function get_import_map(): array {
 		$imports = array();
-		foreach ( $this->get_dependencies( array_unique( $this->queue ) ) as $id => $script_module ) {
-			$imports[ $id ] = $this->get_src( $id );
+		foreach ( $this->get_dependencies( array_unique( $this->queue ) ) as $id ) {
+			$src = $this->get_src( $id );
+			if ( '' !== $src ) {
+				$imports[ $id ] = $src;
+			}
 		}
 		return array( 'imports' => $imports );
 	}
 
 	/**
-	 * Retrieves the list of script modules marked for enqueue.
-	 *
-	 * @since 6.5.0
-	 *
-	 * @return array<string, array> Script modules marked for enqueue, keyed by script module identifier.
-	 */
-	private function get_marked_for_enqueue(): array {
-		return wp_array_slice_assoc(
-			$this->registered,
-			$this->queue
-		);
-	}
```

Comment:
The Query Monitor plugin uses this so this is triggering a fatal error on sites with the plugin installed.

The plugin repo indicates 200K+ installs but I think that count is for production sites only, although I don't know if that is calculated via the environment type setting or excludes localhost, etc. 

Either way, I think it's more than 200K developers it will affect.